operator/src/test/python/test_wdt_mii_filter.py

```diff
@@ -0,0 +1,243 @@
+import os
+import unittest
+import yaml
```

Comment:
Which interpreter are we using here for package yaml?

Reply:
I've been running with 2.7.1 and I believe that's the version running with WLST.

Reply:
I think anyone who tries to build it needs to update the python to install the yaml package.  I got import error - No module named yaml.

Reply:
Thanks.  Any suggestions on alternative?  I know there is a yaml parser in WDT.

Reply:
In addition to `topology.yaml`, we now also have `topology.json`. @ankedia added this to make it easier to consume the topology from scripts. Could you use this and `jq`?

Reply:
I think this is the first time we introduce unit testing python into the test. WDT has it's own yaml parser,  I don't think we want to introduce the dependency between two projects.  If we agreed to build time python requirement, then we can just make it a requirement.  @rjeberhard ?

Reply:
Can the build or test suite verify that you have the right dependency installed? Can we verify that it's possible to build on both Oracle Linux and MacOS?

Reply:
MacOS comes with Python 2.7 by default and it's possible, but a pain, to replace this with the Homebrew install of Python 3. What does this work require?